### PR TITLE
[PAY-1367][PAY-1400][PAY-1401][PAY-1402][PAY-1419] Fix chat tile unfurl bugs

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

- unfurl albums
- remove 1px bottom border on unfurls with no text
- don't scale up/down on tile hover/active
- fix border radius on the chat tiles
- pause chat playlist on mobile when pressed while it's playing

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

local web and mobile dapps v stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

File: packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx [hunk 1]
```diff
@@ -222,23 +217,34 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
...
+      setEmptyUnfurl(false)
     }
   }, [linkValue])
 
-  const chatStyles = !hideMessage
-    ? isAuthor
-      ? { ...styles.unfurl, ...styles.unfurlAuthor }
-      : { ...styles.unfurl, ...styles.unfurlOtherUser }
-    : styles.unfurl
+  const { secondaryDark1, neutralLight7 } = useThemeColors()
```
Comment (dharit-tan) [inline packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx:232]: nit: separate this out into its own function?

File: packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx [hunk 2]
```diff
@@ -222,23 +217,34 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
...
+
+  const borderBottomColor = isAuthor ? secondaryDark1 : neutralLight7
+  const borderBottomWidth = hideMessage
```
Comment (dharit-tan) [inline packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx:235]: what's the difference between borderWidth 0 vs undefined?

File: packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx [hunk 3]
```diff
@@ -178,6 +178,11 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
     )
   }
 
+  const unfurlStyles = {
```
Comment (rickyrombo) [inline packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx:181]: nit: can call cn here and set it to eg `unfurlClassNames` and pass to className directly below

File: packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx [hunk 4]
```diff
@@ -276,21 +282,21 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
                       : null
                   }
                 >
-                  {isPlaylistUrl(linkValue) ? (
+                  {isCollection ? (
                     <ChatMessagePlaylist
```
Comment (dharit-tan) [inline packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx:286]: dang @rickyrombo these components are starting to look mighty similar lol. probs not worth consolidating at this point tho. if it ain't broke don't fix.

File: packages/mobile/src/screens/chat-screen/ChatMessagePlaylist.tsx [hunk 5]
```diff
@@ -106,7 +106,7 @@ export const ChatMessagePlaylist = ({
     }))
   }, [tracks, uidMap])
 
-  const isActive = playingUid !== null && playingUid === uid
+  const isActive = tracksWithUids.find((track) => track.uid === playingUid)
```
Comment (dharit-tan) [inline packages/mobile/src/screens/chat-screen/ChatMessagePlaylist.tsx:109]: nice

File: packages/web/src/pages/chat-page/components/ChatMessagePlaylist.tsx [hunk 6]
```diff
@@ -129,25 +129,24 @@ export const ChatMessagePlaylist = ({
...
-        isLoading={status === Status.LOADING || status === Status.IDLE}
-        isTrending={false}
-        showRankIcon={false}
-        numLoadingSkeletonRows={tracksWithUids.length}
-        togglePlay={() => {}}
-        playingTrackId={playingTrackId}
-        variant='readonly'
-      />
-    </div>
+    // You may wonder why we use the mobile web playlist tile here.
```
Comment (dharit-tan) [inline packages/web/src/pages/chat-page/components/ChatMessagePlaylist.tsx:132]: lol

**Review by dharit-tan — APPROVED** [review]

**Review by rickyrombo — COMMENTED** [review]
I may have an idea on how to simplify this a bit - mind if I branch off and make a PR to your PR?

There's still a few bugs with your impl from what I can see:

Web unfurls by themselves have pink undertones to their corners, and aren't rounded on the bottom left:
image

Web image urls are a bit busted:
image

**Comment by sddioulde** [comment]
as im off today, i quickly addressed some of the comments, and im merging so that the changes are available before next QA on monday. looks like the remaining items are mostly nits too.
@dharit-tan feel free to make a new branch from main to address the web image url bug you're talking about. thx

**Pull request merged**